### PR TITLE
Add WKC logo and info

### PR DESCRIPTION
**Pull request by sirmapy**
This is a follow up on adding of WKC logo and info. Payment already made on the pull request #21275

https://github.com/trustwallet/assets/pull/21275

### CODE REVIEW

**Comment by sirmapy** [comment]
Payment already made on pull request #21275

**Comment by sirmapy** [comment]
@Cryptocool1 Payment already made on pull request https://github.com/trustwallet/assets/pull/21275

**Comment by Cryptocool1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by sirmapy** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
Our token was created by Sir Mapy for SMC DAO as a tutorial token to teach new users about cryptocurrency. We make youtube videos using our tokens thus driving cryptocurrency adoption. A link to one of our youtube videos can be found here. https://www.youtube.com/watch?v=sg5US65dkuo&t=231s

• How is the token being distributed? Are you conducting an airdrop?
The token has already been distributed. There was no airdrop. It was sent to pancakeswap for the general public to buy.

• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
There was no presale or initial coin offering. We did a fair launch on 20 March 2022. All of the coin was created and sent to pancakeswap.finance where liquidity was added. We currently have over 11,000 holders since launch.

• Is the token listed on a centralized/decentralized exchange?
Yes it is listed on Pancakeswap.finance, latoken exchange and cointiger exchange. Here is a link on coinmarketcap: https://coinmarketcap.com/currencies/wiki-cat/markets/

• Is a Smart Contract audit completed? Where can we find this information?
Yes. We have a smart contract audit with solidity.finance 
Here is a link as proof https://solidity.finance/audits/WIKICAT/

File: blockchains/smartchain/assets/0x6Ec90334d89dBdc89E08A133271be3d104128Edb/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "website": "https://wikicatcoin.com/",
+  "description": "Wiki Cat is a tutorial token with unique deflationary mechanism created by Sir Mapy for SMC DAO.",
+  "explorer": "https://bscscan.com/token/0x6Ec90334d89dBdc89E08A133271be3d104128Edb",
+  "type": "BEP20",
+  "symbol": "WKC",
+  "decimals": 18,
+  "status": "active",
+  "id": "0x6Ec90334d89dBdc89E08A133271be3d104128Edb",
+  "tags": [
+           "memes", "Collectibles & NFTs", "Doggone Doggerel"
```
Comment (cryptomanz1) [inline blockchains/smartchain/assets/0x6Ec90334d89dBdc89E08A133271be3d104128Edb/info.json:12]: You need remove ```"Collectibles & NFTs", "Doggone Doggerel"``` from the line

File: blockchains/smartchain/assets/0x6Ec90334d89dBdc89E08A133271be3d104128Edb/info.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "Wiki Cat is a tutorial token with unique deflationary mechanism created by Sir Mapy for SMC DAO.",
+  "explorer": "https://bscscan.com/token/0x6Ec90334d89dBdc89E08A133271be3d104128Edb",
+  "type": "BEP20",
+  "symbol": "WKC",
+  "decimals": 18,
+  "status": "active",
+  "id": "0x6Ec90334d89dBdc89E08A133271be3d104128Edb",
+  "tags": [
+           "memes"
+       ]
```
Comment (cryptomanz1) [inline blockchains/smartchain/assets/0x6Ec90334d89dBdc89E08A133271be3d104128Edb/info.json:13]: and you need put coma after ```]``` this symbol

**Comment by cryptomanz1** [comment]
@sirmapy please update your branch with the recent master

**Comment by sirmapy** [comment]
We didn't get to 13k holders in 4 days. The project have been in operation since March 2022 and we've been able to build a strong community via social media and word of mouth. We have over 13k members on our growing telegram group https://t.me/wikicatcoin and over 9k followers on twitter https://twitter.com/WikiCatCoin. 

We applied to add our logo and info some months back in this pull request https://github.com/trustwallet/assets/pull/21275 but at that time we didn't meet the criteria of 10k holders. Over the months we've worked with a couple of influencers and built our community organically to grow our token holders.

**Comment by sirmapy** [comment]
@Cryptocool1 @cryptomanz1

**Pull request merged**